src/RenderingLists.jsx

```diff
@@ -0,0 +1,32 @@
+const listForRender = [
+    { value: 3, name: "List item value 3" },
+    { value: 4, name: "List item value 4" },
+    { value: 10, name: "List item value 10" },
+    { value: 13, name: "List item value 13" },
+    { value: 62, name: "List item value 62" },
+    { value: 22, name: "List item value 22" },
+    { value: 47, name: "List item value 47" },
+    { value: 7, name: "List item value 7" },
+    { value: 8, name: "List item value 8" },
+  ];
+  
+  export function RenderingLists() {
+    const listItems = listForRender.map(list =>
+    <li>{list.name}</li>
+    );
+    const valueList = listForRender.map(list => {
```

Comment:
You should filter before mapping. You can also chain the filter and map methods like so
```suggestion
    const valueList = listForRender.filter(..do something...).map(...do something...)
```